faqs/galaxy/ES/histories_create_new.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Para la creación de un historial nuevo
+description: Los historiales son una parte importante de Galaxy, la mayoría de la gente utiliza un historial para cada análisis nuevo. Asegúrate siempre de darle buenos nombres a tus historiales, de tal forma que después puedas encontrar fácilmente tus resultados.
+area: historias
```

Comment:
I translated the area as well (please check), since this is only used for the groupings on the overview page (see screenshot in main thread)